app-backend/tile/src/main/scala/image/Mosaic.scala

```diff
@@ -267,8 +267,10 @@ object Mosaic {
           val tiles = maybeTiles.flatten
           if (tiles.nonEmpty)
             Option(tiles.reduce(_ merge _))
-          else
-            Option.empty[MultibandTile]
+          else {
+            val cellType = IntUserDefinedNoDataCellType(0)
+            Option(ArrayMultibandTile.empty(cellType, 3, 256, 256))
+          }
```

Comment:
Sorry about the `alloc`, forgot that didn't exist on IntArrayTile (I think I did that before). The call would be `ofDim`.

`IntArrayTile.ofDim(256, 256).renderPng`

Though what you did works as well.